bin/opencollective.js

```diff
@@ -30,4 +30,9 @@ function printBadge() {
 	console.log("\n");
 }
 
-printBadge();
+function isTrue(value) {
+	return !!value && value !== "0" && value !== "false";
+}
+var envDisable = isTrue(process.env.DISABLE_OPENCOLLECTIVE) || isTrue(process.env.CI);
+
+if (!envDisable) printBadge();
```

Comment:
```suggestion
envDisable?null : printBadge();
```

Reply:
The ternary check is used to return a value, here we don't need to return anything

Reply:
I prefer the statement, is more readable and we don't need an expression :)